HISTORY.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Next Version
-
+* Allow country_code on verify_on_gateway API call [adbatista]
```

Comment:
I don't think we should be changing this file, it looks like [Duff](https://github.com/spreedly/spreedly-gem/commits/master/HISTORY.md) is responsible for merging all the pull requests, bumping the version number and modifying the history file. If you look at the other [pull requests](https://github.com/spreedly/spreedly-gem/pulls?q=is%3Apr+is%3Aclosed), none of them edit this history file, so we should probably leave it alone.